transport/http/handler.go

```diff
@@ -50,6 +51,11 @@ type handler struct {
 func (h handler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	start := time.Now()
 
+	if req.URL != nil && strings.HasPrefix(req.URL.Path, "/debug/") {
```

Comment:
- Debug pages are attached to the default HTTP server mux (something that can be changed later).
 - Effectively right now you can start the default HTTP server, or implement any forwarding to the default server mux.
 - Attaching automatically to HTTP inbound avoids requiring an extra port, that need to be explicitly configured etc.